hadoop-cloud-storage-project/pom.xml

```diff
@@ -34,6 +34,7 @@
     <module>hadoop-cos</module>
     <module>hadoop-huaweicloud</module>
     <module>hadoop-tos</module>
+    <module>hadoop-cloud-storage-dist</module>
```

Comment:
It seems like we would never need to enter execution of `hadoop-cloud-storage-dist` unless we are building a distro (activating `-Pdist`). Should we also wrap inclusion of the sub-module here behind activation of the `dist` profile?